### ISSUE TITLE
API stability

### ISSUE THREAD

**Issue by alapidas** [slack-go/slack#107]
How stable is the current API?  `v0.0.1` is over a year old at this point - will the current code be tagged at some point soon?

Thanks

**Comment by nlopes** [comment]
Yes, I am currently doing a batch of changes and will tag this soon(ish).

**Comment by tommiv** [comment]
It's > 2017 now. What should I choose for a new project? I started with `glide get github.com/nlopes/slack`, copypaste code snippet from readme and immediately encountered a build error. Most of the examples are broken as well, so it seems that the new codebase is not synced with docs (examples). So I dig into sources now, but I'm not sure what is the right way to establish realtime websocket connection and handle incoming messages.

**UPD**: I think I got it, glide has installed a 0.0.1 version regardless what I answer about which version to install. I replaced the commit hash in glide.lock to the latest commit in HEAD and now it works. But I still think that release tags is a good idea.
